lua/telescope/make_entry.lua

```diff
@@ -794,7 +794,7 @@ function make_entry.gen_from_registers(opts)
   return function(entry)
     return make_entry.set_default_entry_mt({
       value = entry,
-      ordinal = entry,
+      ordinal = vim.fn.getreg(entry),
```

Comment:
lets only do `vim.fn.getreg` once. can you move the result in a `local v` and use that for ordinal as well as content? thanks

Reply:
Yeah that makes sense.